packages/tasit-contracts/3rd-parties/decentraland/scripts/mana/truffle.js

```diff
@@ -0,0 +1,37 @@
+require("babel-register");
+require("babel-polyfill");
+
+const HDWalletProvider = require("truffle-hdwallet-provider");
+
+const mnemonic = process.env.MNEMONIC;
+const token = process.env.INFURA_TOKEN;
+
+module.exports = {
+  networks: {
+    development: {
+      host: "localhost",
```

Comment:
This file is named `truffle.js`, whereas the one for land is named `truffle-config.js`

Reply:
I've used the same file name that is used on the decentraland `truffle` projects.

Reply:
Ah ok. Fair enough. Resolving this.